src/components/tiles/table/use-grid-context.ts

```diff
@@ -75,9 +80,11 @@ export const useGridContext = ({ modelId, showRowLabels, triggerColumnChange }:
 
   // called by ReactDataGrid when selected rows change
   const onSelectedRowsChange = useCallback((_rows: Set<React.Key>) => {
+    clearSelection({ row: false });
     _rows.delete(inputRowId.current);
-    sharedSelection.setSelected(modelId, Array.from(_rows) as string[]);
-  }, [modelId, sharedSelection]);
+    dataSet.setSelectedCases(Array.from(_rows) as string[]);
+    triggerColumnChange();
```

Comment:
Like the others above, if this is necessary it should be `triggerRowChange()`, but it may not be necessary at all. This is handling the case of a selection which started with the table so RDG already knows exactly which rows have changed and so I would think that it wouldn't be necessary to trigger an external re-render of the table.